core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -415,6 +415,7 @@ oppia.directive('conversationSkin', [
             AutogeneratedAudioPlayerService.cancel();
             var index = PlayerPositionService.getActiveCardIndex();
             $scope.activeCard = PlayerTranscriptService.getCard(index);
+            $location.hash(null);
```

Comment:
Does making it an empty string help?

Reply:
I did that to prevent any future issues that may happen due to a hash in the url and if it is null, I saw it didnt interfere with anything.